### PR TITLE
porting pblintd init pot function

### PR DESCRIPTION
**Pull request by xyuan**
porting pblintd init pot function

[BFB]

### CODE REVIEW

File: components/scream/src/physics/shoc/shoc_pblintd_init_pot.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include "shoc_pblintd_init_pot_impl.hpp"
+#include "ekat/scream_types.hpp"
+
+namespace scream {
+namespace shoc {
+
+/*
+ *  * Explicit instantiation for doing ice melting on Reals using the
```
Comment (jeff-cohere) [inline components/scream/src/physics/shoc/shoc_pblintd_init_pot.cpp:8]: Delete "ice melting" to make this generic.

Comment (xyuan) [reply components/scream/src/physics/shoc/shoc_pblintd_init_pot.cpp]: yes, done!

**Review by jgfouca — CHANGES_REQUESTED** [review]
Please upstream-merge this PR and adopt the latest code conventions.

**Comment by xyuan** [comment]
I am closing this PR

**Pull request closed without merging**